Demo/Sources/Players/PlaybackView.swift

```diff
@@ -200,11 +204,19 @@ private struct MainView: View {
         .menuOrder(.fixed)
     }
 
+    @ViewBuilder
+    private func artwork(for image: UIImage) -> some View {
+        Image(uiImage: image)
+            .resizable()
+            .aspectRatio(contentMode: .fit)
+            .frame(maxWidth: .infinity, maxHeight: .infinity)
+    }
+
     @ViewBuilder
     private func video() -> some View {
         ZStack {
-            if player.mediaType == .audio {
-                image(name: "music.note.tv.fill")
+            if player.mediaType == .audio, let image {
+               artwork(for: image)
```

Comment:
```suggestion
				if player.mediaType == .audio {
					if let image {
						artwork(for: image)
					}
					else {
						image(name: "music.note.tv.fill")					}
```

Reply:
I had initially written the code this way, but I thought the result was better without the music note placeholder after testing it.

We can of course keep it if you think the result is better.

Reply:
Without placeholder is also fine.